src/main/java/org/skriptlang/skript/lang/util/SkriptQueue.java

```diff
@@ -0,0 +1,85 @@
+package org.skriptlang.skript.lang.util;
+
+import ch.njol.yggdrasil.YggdrasilSerializable;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.*;
+
+/**
+ * A queue of elements.
+ * This is designed not to support null elements, i.e. putting in nothing should do nothing.
+ */
+public class SkriptQueue extends LinkedList<@NotNull Object>
+	implements Deque<Object>, Queue<Object>, YggdrasilSerializable {
+
+	@Override
+	public boolean add(Object element) {
+		if (element == null)
+			return false;
```

Comment:
So the always-true part comes from the fact that lists, queues, stacks, etc. will _always_ be updated when you add stuff to them (because there isn't a case where the thing just doesn't get added) whereas this doesn't support some kinds of elements so if you try and add Nothing, you won't have modified the collection.
I expected this to be something that happens a lot and I think it would be a little silly just to throw an exception that we then catch and silence during the adding, so it seems easiest just to not throw one at all.